README.md

```diff
@@ -8,6 +8,8 @@ It also allows deleting a corrupt, incomplete or dangling cache. A cache can be
 
 This extension builds on top of [cache management](https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows#managing-caches) capabilities exposed by GitHub [APIs](https://docs.github.com/en/rest/actions/cache).
 
+**Note:** This extension currently supports github.com repositories only and will start supporting GHES (GitHub Enterprise Server) by the next ship version i.e. 3.7.
```

Comment:
```suggestion
**Note:** This extension currently supports github.com repositories only and will start supporting GHES (GitHub Enterprise Server) starting GHES 3.7.
```

Reply:
Removed this as we don't want to state the version as of now.